core/sdk/src/utils/logger.rs

```diff
@@ -121,7 +114,17 @@ pub fn init_and_configure_logger(version: &str, name: String) -> Result<(), fern
         .level_for("tungstenite", log::LevelFilter::Off)
         .level_for("soketto", log::LevelFilter::Off)
         .level_for("tracing", log::LevelFilter::Off)
-        .chain(std::io::stdout())
-        .apply()?;
+        .chain(std::io::stdout());
```

Comment:
Don't you have to call `chain()` after `if let Some(modules) = additional_modules {`?